Model/Message.php

```diff
@@ -265,7 +271,10 @@ public function merge(Message $message)
             $this->addSource($source);
         }
 
-        $this->new = $message->isNew();
+        $this->setNew($message->isNew());
+        if ($localeString = $message->getLocaleString()) {
+            $this->localeString = $localeString;
+        }
```

Comment:
This if statement will always be true. 
Correct me if I'm wrong but does it bring any value? We have the functionality just a couple of lines above...